index.js

```diff
@@ -32,6 +32,38 @@ function getMainArgs() {
   return process.argv.slice(2);
 }
 
+function setOptionValue(parseOptions, option, value, result) {
+  const multiple = parseOptions.multiples &&
+    parseOptions.multiples.includes(option);
```

Comment:
We should probably try to keep this package robust, primordials-style.

Reply:
So not trusting that `parseOptions.multiples` has an `includes` that works as intended?

I'm open to that discussion, but could perhaps be a separate issue? (Partly because I only moved the code, and partly because my PR already has quite a few changes.)

Reply:
Added #31